Avoid an assert while printing on Windows in Debug mode
---
diff --git a/src/3rdparty/chromium/printing/page_range.cc b/src/3rdparty/chromium/printing/page_range.cc
index 1d5c59fe7b69e2..3bb7fc232b07b3 100644
--- a/src/3rdparty/chromium/printing/page_range.cc
+++ b/src/3rdparty/chromium/printing/page_range.cc
@@ -13,6 +13,10 @@ namespace printing {
 
 // static
 void PageRange::Normalize(PageRanges& ranges) {
+  if (ranges.empty()) {
+    return;
+  }
+
   std::sort(ranges.begin(), ranges.end());
   PageRanges::iterator dst = ranges.begin();
   for (PageRanges::iterator src = ranges.begin() + 1; src < ranges.end();
